spec/controllers/community_controller_spec.rb

```diff
@@ -36,7 +36,8 @@
         user_opted_out = create(:user, hide_email: true)
         get :index
         expect(response.body).to include user.email
-        expect(response.body).not_to include user_opted_out.email
+        # Should this be tested in a controller test?
+        expect(response.body).not_to include user_opted_out.email # false negative; also passes with hide_email: false
```

Comment:
@F3PiX I may be missing something here, but I _think_ this may not be a false negative. Changing `user_opted_out` to have `hide_email: false` will make the test fail.